### PR TITLE
287 create event by user

### LINKED ISSUE #287: [IMPROVEMENT] - Any signed in user should be able to create an event
**Issue Description**
Currently only admins can create an event. It should be possible for any registered user to create an event.
However, we need to ensure that:
- We store the user that has created the event: migration on `events` table
- Only the user that created an event can delete it
- Admins can delete any events

**Desired Behavior**
...

**Acceptance Criteria**
...

**Additional Considerations**
...

**Resources**
...

**Impact**
...

**Related Issues**
...

### PR DESCRIPTION
**Pull request by christinasugiono**
closes #287

### CODE REVIEW

File: .gitignore
Comment (alexstan67) [inline .gitignore:1]: I think it is not linked with this feature

File: db/migrate/20240529022452_add_user_to_events.rb
Comment (alexstan67) [inline db/migrate/20240529022452_add_user_to_events.rb:1]: Is it not better to merge it with `db/migrate/20240530102406_update_events_with_user_id.rb` ?

**Review by alexstan67 — COMMENTED** [review]
I have made some modifications:
- deleted the `.gitignore` entry related to geoip
- Added on the event index view the user that created the event.

:warning: However, we have an eager loading problem I can't solve. I tried `.includes(%i[airport, user])` in the events controller, **without success**. It is annoying, as it pops up every time we load the start page.
Maybe you have another idea? If you manage to solve it, you can merge to `master` :tada:

File: .gitignore
Comment (christinasugiono) [reply .gitignore:1]: Yes I agree... I thought I added it on the analytics tool PR, but for some reason its not there.. and I need it to make sure the `db/geo` is not pushed yet until we're sure its okay to publish the data 😅 So I need to add it to the `.gitignore`

**Pull request merged**